### PR TITLE
Parser Refactor

### PR DESCRIPTION
**Pull request by JulianToledano**
:link: [zboto Link](https://app.clickup.com/t/867759np6)

### CODE REVIEW

File: tools/constants.go [hunk 1]
```diff
@@ -60,6 +65,15 @@ var SupportedOperations = map[string]bool{
...
 	"RemoveVerifier":         true, // MethodsVerifiedRegistry
+	"InvokeContract":         true, // MethodsEVM
+	"GetBytecode":            true, // MethodsEVM
+	"GetStorageAt":           true, // MethodsEVM
+	"InvokeContractReadOnly": true, // MethodsEVM
+	"InvokeContractDelegate": true, // MethodsEVM
+	"ChangeMultiaddrs":       true,
+	"ChangePeerID":           true,
+	"ExtendSectorExpiration": true,
+	"PublishStorageDeals":    true,
```
Comment (raynaudoe) [inline tools/constants.go:76]: Could you please add to which actors these methods belongs to ?

File: tools/parser/helpers.go [hunk 2]
```diff
@@ -132,3 +135,62 @@ func ParseMsigParams(msg *filTypes.Message, height int64, key filTypes.TipSetKey
...
+	return map[string]interface{}{}, errUnknownMethod
+}
+
+func (p *Parser) parseSend(msg *filTypes.Message) map[string]interface{} {
+	metadata := make(map[string]interface{})
+	metadata[tools.ParamsKey] = msg.Params
+	return metadata
+}
+
+//func (p *Parser) parseExec(msg *filTypes.Message, msgRct *filTypes.MessageReceipt,
```
Comment (raynaudoe) [inline tools/parser/helpers.go:157]: Do you need an example to test this one? or why is commented out ?

Comment (JulianToledano) [reply tools/parser/helpers.go:157]: oh, my bad there. That function has been replaced for this one https://github.com/Zondax/filecoin-indexing-rosetta-proxy/pull/32/files#diff-b70287623d9bc6cf64324eefa7111cb2bd90498b1b6160c2cbcbea95a1d28110R37-R62

**Comment by raynaudoe** [comment]
Seems there are some lint complains, you can use the consts in here to replace the actors names
/Users/eze/go/pkg/mod/github.com/filecoin-project/go-state-types@v0.10.0-alpha-9.0.20230118014411-cf80be872456/manifest/manifest.go

**Comment by raynaudoe** [comment]
> Constructor

I think not ... you can add those 2 consts in a const.go file, or just add the lint ignore for those ...

**Comment by JulianToledano** [comment]
🥲 I'll add all methods as constants, don't like the idea of having some as const and others as literals

**Review by raynaudoe — APPROVED** [review]
Nice job! 🚀

**Pull request merged**